### PR TITLE
Put akka-stream-testkit in test configuration (#401)

### PR DESCRIPTION
**Pull request by hseeberger**

### CODE REVIEW

**Comment by ktoso** [comment]
Refs #401

File: project/Dependencies.scala [hunk 1]
```diff
@@ -70,7 +69,8 @@ object Dependencies {
     }
 
     object Test {
-      val akkaTestkit  = "com.typesafe.akka"          %% "akka-testkit"                 % akkaVersion        % "test" // Apache v2
+      val akkaTestkit       = "com.typesafe.akka"      %% "akka-testkit"                % akkaVersion        % "test" // Apache v2
+      val akkaStreamTestkit = "com.typesafe.akka"      %% "akka-stream-testkit"         % akkaVersion        % "test" // Apache v2
```
Comment (ktoso) [inline project/Dependencies.scala]: thanks

**Comment by ktoso** [comment]
LGTM, thanks!

**Comment by jrudolph** [comment]
Thanks, @hseeberger, for cleaning up after me. Cannot remember if I thought anything when putting it that way.

**Comment by ktoso** [comment]
Thanks @hseeberger !

**Comment by hseeberger** [comment]
Sure thing! Finally on the contributors list again ;-)

**Comment by ktoso** [comment]
We missed you, good to have you back on there ;-)

**Pull request merged**